ztp/gitops-subscriptions/argocd/example/siteconfig/example-sno.yaml

```diff
@@ -14,6 +14,11 @@ spec:
   clusters:
   - clusterName: "example-sno"
     networkType: "OVNKubernetes"
+    # installConfigOverrides is a generic way of passing install-config
+    # parameters through the siteConfig.  The capabilities field configures the
+    # composable openshift feature.  In this example, we remove all but the
+    # marketplace component from the optional set of components.
```

Comment:
Minor nit, this is no longer an example. Maybe something like "This default value removes all but the ...."